{
  boolean caughtException=false;
  try {
    client.createNodesInGroup(this.group,1);
  }
 catch (  Exception e) {
    caughtException=true;
  }
  assertTrue(caughtException,"expected an exception to be thrown");
}
